lib/elastic_apm/stacktrace_builder.rb

```diff
@@ -12,7 +12,8 @@ class StacktraceBuilder
     RUBY_VERS_REGEX = %r{ruby(/gems)?[-/](\d+\.)+\d}.freeze
     JRUBY_ORG_REGEX = %r{org/jruby}.freeze
 
-    GEMS_PATH = defined?(Bundler) ? Bundler.bundle_path.to_s : Gem.dir
+    GEMS_PATH = (defined?(Bundler) && Bundler.default_bundle_dir) ?
+                  Bundler.bundle_path.to_s : Gem.dir
```

Comment:
Suggestion for readability

```suggestion
    GEMS_PATH =
      if defined?(Bundler) && Bundler.default_bundle_dir
        Bundler.bundle_path.to_s
      else
        Gem.dir
      end
```

Reply:
I think you force pushed over this?

Reply:
No, not on purpose. I rebased on master and forced pushed, not realizing that the committed suggestion hadn't been pulled down locally.

Reply:
That was my guess. Classic 😊